.github/workflows/pytest.yml

```diff
@@ -0,0 +1,51 @@
+# .github/workflows/app.yaml
+name: Pytest
+
+on: [push]
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["2.7", "3.7"]
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Poetry
+        run: |
+          python -m pip install --upgrade poetry wheel
+      - name: Install dependencies
+        run: |
+          poetry install
+      - name: Run tests without coverage
+        if: ${{ matrix.python-version == 2.7 }}
+        run: |
+          poetry run pytest tests
+      - name: Run tests with coverage
+        if: ${{ matrix.python-version == 3.7 }}
+        run: |
+          poetry run pytest tests --cov-report=term-missing:skip-covered --cov-report=xml --cov=flowpipe | tee pytest-coverage.txt
+      - name: Pytest coverage comment
+        uses: MishaKav/pytest-coverage-comment@main
+        if: ${{ matrix.python-version == 3.7 }}
+        id: coverageComment
+        with:
+          hide-comment: ${{ github.ref == 'refs/heads/master' }}
+          pytest-coverage-path: ./pytest-coverage.txt
+      - name: Update Readme with Coverage Html
+        if: ${{ github.ref == 'refs/heads/master' && matrix.python-version == 3.7 }}
+        run: |
+          sed -i '/<!-- Pytest Coverage Comment:Begin -->/,/<!-- Pytest Coverage Comment:End -->/c\<!-- Pytest Coverage Comment:Begin -->\n\${{ steps.coverageComment.outputs.coverageHtml }}\n${{ steps.coverageComment.outputs.summaryReport }}\n<!-- Pytest Coverage Comment:End -->' ./README.md
+      - name: Commit & Push changes to Readme
+        if: ${{ github.ref == 'refs/heads/master' && matrix.python-version == 3.7 }}
+        uses: actions-js/push@master
+        with:
+          message: Update coverage on Readme
+          branch: 160-replace-travis
```

Comment:
I think this will have to be updated to master before merging, right?

Reply:
Right, thanks for catching this